tests/phpunit/CRM/Utils/Migrate/ImportExportTest.php

```diff
@@ -137,7 +137,7 @@ public function basicXmlTestCases() {
       // CustomGroup params
       array(
         'extends' => 'Activity',
-        'extends_entity_column_value' => array(array_search('Meeting', CRM_Core_PseudoConstant::activityType())),
+        'extends_entity_column_value' => [1],
```

Comment:
Hardcoded?

Reply:
yes - because it's in the dataprovider (I guess) this test won't run in isolation if it is accessing the DB